pkg/kubelet/server/server_test.go

```diff
@@ -1550,25 +1551,16 @@ func TestDebuggingDisabledHandlers(t *testing.T) {
 	paths := []string{
 		"/run", "/exec", "/attach", "/portForward", "/containerLogs", "/runningpods",
 		"/run/", "/exec/", "/attach/", "/portForward/", "/containerLogs/", "/runningpods/",
-		"/run/xxx", "/exec/xxx", "/attach/xxx", "/debug/pprof/profile", "/logs/kubelet.log",
+		"/run/xxx", "/exec/xxx", "/attach/xxx", "/debug/pprof/profile",
```

Comment:
it looks like the changes from this point on in this file are not needed, right?

Reply:
hmm actually these changes are part of unit tests that verifies that error messages returned for endpoints that are disabled are valid. For eg, when /logs is disbaled I expect the error message to be "logs endpoint is disabled". So this will help in future when we move other endpoints out from debuggingHandlers

Reply:
I see, it was just the message that changed